epochStart/metachain/economics.go

```diff
@@ -135,7 +135,7 @@ func (e *economics) ComputeEndOfEpochEconomics(
 	newTokens := big.NewInt(0).Sub(totalRewardsToBeDistributed, metaBlock.AccumulatedFeesInEpoch)
 	if newTokens.Cmp(big.NewInt(0)) < 0 {
 		newTokens = big.NewInt(0)
-		totalRewardsToBeDistributed = big.NewInt(0).Set(metaBlock.AccumulatedFeesInEpoch)
+		totalRewardsToBeDistributed = big.NewInt(0).Sub(metaBlock.AccumulatedFeesInEpoch, metaBlock.DevFeesInEpoch)
```

Comment:
I don't think this should be subtracted here, as it is already adjusted below.
```
remainingToBeDistributed := big.NewInt(0).Sub(totalRewardsToBeDistributed, metaBlock.DevFeesInEpoch)
```

As far as I remember totalRewardsToBeDistributed was supposed to contain also the developer fees

Reply:
yes, removed